lib/hacluster.pm

```diff
@@ -320,6 +320,10 @@ sub ha_export_logs {
     # HAWK test logs if present
     upload_logs("/tmp/hawk_test.log", failok => 1);
     upload_logs("/tmp/hawk_test.ret", failok => 1);
+
+    # HANA hdbnsutil logs
```

Comment:
I'm OK to add this in the HA library if @alvarocarvajald agree too, but please condition this to HANA test. There is a `HANA` variable, it should be easy, no? ;-)

Reply:
Done